app/javascript/providers/GeneSearchProvider.js

```diff
@@ -108,6 +103,13 @@ export function PropsGeneSearchProvider(props) {
   )
 }
 
+export function buildParamsFromQuery(query) {
+  const queryParams = queryString.parse(query)
+  return {
+    page: queryParams.genePage ? parseInt(queryParams.genePage) : 1,
+    genes: queryParams.genes ? queryParams.genes : ''
+  }
```

Comment:
I notice `preset` is not here, but is [in `buildParamsFromQuery` for `StudySearchProvider`](https://github.com/broadinstitute/single_cell_portal_core/pull/627/files#diff-204721323bf02fa877c76e6f6754dc5dR153).

Is `preset` needed here to only return COVID-19 studies from gene search in the COVID-19 space?

Reply:
This is a great catch.  I'll add preset here.  It's not necessary right now because we decided to not have gene search on the covid 19 space, but I suspect soon we will want to, and this will stop a future bug